enhancements/sig-architecture/123-addon-multiple-configs/README.md

```diff
@@ -0,0 +1,230 @@
+# Addon Multiple Configs with Same GVK
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+This proposal proposes an enhancement to the addon configurations, enabling users to configure multiple configs of the same GVK.
+
+## Motivation
+
+Users have three locations to set [addon configurations](https://open-cluster-management.io/concepts/addon/#add-on-configurations): 
+1. In `ClusterManagementAddon` `spec.supportedConfigs`, user can set the `defaultConfig` where all the addons to have same configurations.
+2. In `ClusterManagementAddOn`, `spec.installStrategy.placements[].configs`, user can set configs for addons on a group of clusters selected by placement.
+3. In `ManagedClusterAddOn` `spec.configs`, user can set configs for addon on a specific cluster.
+
+Currently, configs with the same GVK only support a single config. The config with the same GVK in place `3` will override the config in place `2`, 
+which will override the config in place `1`. The final effective addon configs are listed in the mca.status.
+
+Now we are seeing the requirement to set multiple configs for the same GVK. For example, for `OpenTelemetryCollector`, one CR might be configured to 
+collect traces and another to collect logs on the same cluster. This proposal aims to satisfy that requirement.
+
+### Goals
+
+- Enhance `ClusterManagementAddon` and `ManagementClusterAddon` API to allow users to configure multiple addon configs with the same GVK.
+
+### Non-Goals
+
+- `ClusterManagementAddon` `spec.supportedConfigs` does not allow user to configure mutiple addon configs with same GVK in the `defaultConfig`.
+
+## Proposal
+
+### User Stories
+
+1. The admin can configure the `ClusterManagementAddon` API to set mutilple configs with same gvk per install strategy.
+2. The admin can configure the `ManagementClusterAddon` API to set mutilple configs with same gvk for specific cluster.
+
+## Design Details
+
+### API changes
+
+`cma.spec.installStrategy.placements[].configs` and `mca.spec.configs` already support mutiple configs. 
```

Comment:
Multiple configurations are supported but since the GVK is used as the identifier, only one is persisted, right?

Reply:
yes, today user can define multiple configs with the same gvk at the API level, but only one will be listed in the mca.status.